pkg/migrations/mls/20240528181851_init-schema.up.sql

```diff
@@ -0,0 +1,71 @@
+SET statement_timeout = 0;
+
+--bun:split
+CREATE TABLE installations(
+	id BYTEA PRIMARY KEY,
+	created_at BIGINT NOT NULL,
+	updated_at BIGINT NOT NULL,
+	inbox_id BYTEA NOT NULL,
+	key_package BYTEA NOT NULL,
+	expiration BIGINT NOT NULL
+);
+
+--bun:split
+CREATE TABLE group_messages(
+	id BIGSERIAL PRIMARY KEY,
+	created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+	group_id BYTEA NOT NULL,
+	data BYTEA NOT NULL,
+	group_id_data_hash BYTEA NOT NULL
+);
+
+--bun:split
+CREATE INDEX idx_group_messages_group_id_id ON group_messages(group_id, id);
+
+--bun:split
+CREATE UNIQUE INDEX idx_group_messages_group_id_data_hash ON group_messages(group_id_data_hash);
+
+--bun:split
+CREATE TABLE welcome_messages(
+	id BIGSERIAL PRIMARY KEY,
+	created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+	installation_key BYTEA NOT NULL,
+	data BYTEA NOT NULL,
+	hpke_public_key BYTEA NOT NULL,
+	installation_key_data_hash BYTEA NOT NULL
+);
+
+--bun:split
+CREATE INDEX idx_welcome_messages_installation_key_id ON welcome_messages(installation_key, id);
+
+--bun:split
+CREATE UNIQUE INDEX idx_welcome_messages_group_key_data_hash ON welcome_messages(installation_key_data_hash);
+
+--bun:split
+CREATE TABLE inbox_log(
+	sequence_id BIGSERIAL PRIMARY KEY,
+	inbox_id BYTEA NOT NULL,
```

Comment:
`inbox_id` is now a BYTEA field in the DB.

This will mean that any `inbox_id` values that get sent over the wire will need to be hex decodable. If you send something that isn't composed of hex characters it will error.

Reply:
Would it make sense to change inbox_id type from String to Vec<u8> so that we don't need to handle encoding/decoding?

Reply:
Nobody else wants to deal with byte arrays, so we are handling everything as close to the db as possible.

Reply:
The frustration was that the platform SDKs were converting our byte arrays to strings immediately anyways, and then had to re-decode to pass them back in to our SDK as arguments. Having it be all strings in libxmtp just made it nice and consistent with the platform SDK. 

I actually agree with you. I like byte arrays. But the people have spoken, and we built all of libxmtp around inbox_id being a string. Would be a pain to change direction now, so we are only converting to a byte array where it matters most (shrinking the size of our DB indexes).

Reply:
That makes sense, thanks!